js/ticket.js

```diff
@@ -0,0 +1,88 @@
+let place = [];
+let price = [];
+function gettingID(clickID,event){
+
+    let sliceTo = event.target.className.slice(10, 12);
```

Comment:
as I got it right, you want to get the number of the selected seat. If it's right then change this part to a more generic way. Slicing with just a hardcoded index is not safe. For example you can try to first find the index of number, and starting slice from there